src/core/osd.c

```diff
@@ -61,10 +95,14 @@ void osd_llock_show(bool bShow) {
         return;
     }
 
-    if (g_latency_locked)
+    if (g_latency_locked){
```

Comment:
couple of code-formatting errors, please use vscode's `format document` function.
that will format the code according to the clang-format file.

Reply:
Done with new commit.